examples/plotting/wcsaxes_map_example.py

```diff
@@ -28,19 +28,40 @@
 # command expects pixel coordinates. It is also possible to plot a point using the
 # map coordinates if we pass the transformation but this transform expects
 # the values to be in degrees and not arcseconds.
+
+# sphinx_gallery_defer_figures
+
 fig = plt.figure()
 ax = plt.subplot(projection=smap)
-print(ax)
+
 ax.imshow(smap.data)
+
+################################################################################
+# Add an overlay grid, showing the native coordinates of the map.
+
+# sphinx_gallery_defer_figures
+
+ax.coords.grid(color='yellow', linestyle='solid', alpha=0.5)
+
+################################################################################
 # plot a point in the middle of the image
+
+# sphinx_gallery_defer_figures
+
 pixel_coord = [smap.data.shape[0]/2., smap.data.shape[1]/2.] * u.pix
 ax.plot(pixel_coord[0], pixel_coord[1], 'x', color='white',
         label=f'Pixel coordinate [{pixel_coord[0]}, {pixel_coord[1]}]')
-ax.coords.grid(color='yellow', linestyle='solid', alpha=0.5)
 
-map_coord = ([-300, 200] * u.arcsec)
+
+################################################################################
 # Using the transform command expects coordinates in degrees and not arcseconds
+
+# sphinx_gallery_defer_figures
+
```

Comment:
```suggestion
```
Don't defer a figure in the last code block of an example.  You pushed it onto the first code block of the next example!